### ISSUE TITLE
[REQUEST] Add option to config to control log level

### ISSUE THREAD

**Issue by ubulem** [NECROBOTIO/NecroBot#5036]
Would be nice to have opportunity to filtrate info messages in command line by setting log level, e.g. WARNING/INFO/DEBUG etc.
For example I wanna see only catches or transfers and and tired of Sniper or Pokestops messages.
Or even control messages tags, like show only logs with tags "Catch". 
That would be great.

**Comment by theloser** [comment]
I totally love this idea! (Right now I am just compiling the source after I deleted all those messages I don't want to be bothered with :-))

**Comment by repelsteeltjehd** [comment]
that is possible by editing the config file

**Comment by burntable** [comment]
How so?

**Comment by theloser** [comment]
I, too, can't find any settings in the default config.json which seems to control the verbosity of the bot. Maybe it's an undocumented settings, and if so, what's the setting called?

**Comment by repelsteeltjehd** [comment]
i know the code is somewere.. i must be called (POKESTOP), i've managed to get the recycling message on quiet.. and (INFO) disabled :)

the code you are looking for is not implented in the config file.

**Comment by burntable** [comment]
Well yeah, that's all you can filter with the config right now, I think.
OP wants a feature that let you filter all kinds of messages. For example only show warnings.

**Comment by nfettinger** [comment]
I dont believe it would be possible to do this. The log level is set up BEFORE the config settings are entered in, in case there is an issue with the parsing. We may be able to change it after the fact, but this would be a low priority issue. Instead, build from source with the flag changed.

**Comment by Andrerm124** [comment]
I'd like to have a look into setting this up actually. No promises but later today (After I eat) I might have something to show for it

**Comment by tatejensen6** [comment]
Does anyone know a amazing config for necrobot?? Having trouble with this bot

**Comment by Andrerm124** [comment]
Going to delay this until a later date (I've put it in my todo list)
